### ISSUE TITLE
Noto font + styles / script wrongly loaded in front

### ISSUE THREAD

**Issue by maximebj** [ahmadawais/create-guten-block#246]
Hi !

I've noticed that some scripts, styles and fonts (Noto) are loaded in front but not needed. I have the issue in several projects with CGB, resulting in useless HTTP calls.

```
<link rel='stylesheet' id='wp-components-css'  href='https://xyoos.local/wp-includes/css/dist/components/style.min.css?ver=5.2.4' type='text/css' media='all' />
<link rel='stylesheet' id='wp-editor-font-css'  href='https://fonts.googleapis.com/css?family=Noto+Serif%3A400%2C400i%2C700%2C700i&#038;ver=5.2.4' type='text/css' media='all' />
<link rel='stylesheet' id='wp-block-editor-css'  href='https://xyoos.local/wp-includes/css/dist/block-editor/style.min.css?ver=5.2.4' type='text/css' media='all' />
<link rel='stylesheet' id='wp-nux-css'  href='https://xyoos.local/wp-includes/css/dist/nux/style.min.css?ver=5.2.4' type='text/css' media='all' />
<link rel='stylesheet' id='wp-editor-css'  href='https://xyoos.local/wp-includes/css/dist/editor/style.min.css?ver=5.2.4' type='text/css' media='all' />
```

It comes from the `register_block_type()` in the `init` hook.

Suggestion : put this in a separate hook, only for admin, and limit to `post.php` pages only. Something like that : 

```
function cgb_admin_assets() {
		
	$screen = get_current_screen();
	if ( ! $screen->base == 'post' ) {
    	    return;
	}

	register_block_type(
		'cgb/block-xyoos-blocks', array(	
			'style'         => 'xyoos_blocks-cgb-style-css',
			'editor_script' => 'xyoos_blocks-cgb-block-js',
			'editor_style'  => 'xyoos_blocks-cgb-block-editor-css',
		)
	);
}
add_action( 'current_screen', 'cgb_admin_assets' );
``` 

What do you think? Am I missing something?

Thanks.

**Comment by maximebj** [comment]
I've noticed that the old way worked better (with enqueue_block_editor_assets and enqueue_block_assets hooks).

**Comment by ahmadawais** [comment]
This issue must have been solved via #267. 

Successfully published:
 - cgb-scripts@1.23.0
 - create-guten-block@1.14.0

You do have to manually update the [init.php file like this](https://github.com/ahmadawais/create-guten-block/pull/267/files).
